clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -229,6 +229,41 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
     ],
   );
 
+  const handleAcceptAll = useCallback(() => {
```

Comment:
bumped these up to parent to take advantage of them at this level for the automated preference